schema.sql

```diff
@@ -5,5 +5,17 @@ CREATE TABLE animals(
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL(18,2),
-    species VARCHAR
+    species_id INT FOREIGN KEY,
+    owners_id INT FOREIGN KEY
+);
```

Comment:
- [ ] [REQUIRED] Kindly note that the project requirement states that you should modify the animals table. I can see that you just inserted the column `species_id` and the column `owner_id` into an existing table which is not correct. To modify the `animals` table, you need to use the ALTER TABLE command. This command will help you modify an existing table. Take a look at [this](https://www.w3schools.com/sql/sql_alter.asp) for more insight.

Reply:
Thank you @Whoistolu  for the great Review and the suggested documentation. I did add it before but I will make it more clear. 🙏